asset_dashboard/static/js/components/BaseTable.js

```diff
@@ -128,24 +91,26 @@ const BaseTable = ({ rows = [],  getTrProps = props => props, rowClassNames, sel
             </li>
           </ul>
           
-          <ul className="pagination col d-flex justify-content-end">
-            {[15, 30, 45].map(pSize => (
-              <li 
-                key={pSize} 
-                value={pSize} 
-                className="page-item ml-2"  
-                >
-                <button 
-                  onClick={e => {
-                    setPageSize(Number(pSize))
-                  }}
-                  className={`btn btn-light btn-sm ${(pSize == pageSize) ? 'active' : '' }`}
+          {pageSizeIncrements.length > 1 &&
```

Comment:
hide the increment buttons if there is only one item in the list (like in the asset search table, since that UI doesn't have much space to take up when considering the layout of the entire page.)